src/core/unit.h

```diff
@@ -584,6 +585,10 @@ typedef struct UnitVTable {
 
         bool (*can_reload)(Unit *u);
 
+        /* Add a bind/image mount into the unit namespace while it is running. */
+        int (*live_mount)(Unit *u, const char *src, const char *dst, sd_bus_message *message, MountInNamespaceFlags flags, const MountOptions *options, sd_bus_error *error);
+        int (*can_live_mount)(Unit *u, sd_bus_error *error);
```

Comment:
For new `can_*` funcs let's make `u` const from day one (this has been the case for `can_freeze`)